client/src/main/java/io/getstream/chat/android/client/ChatClient.kt

```diff
@@ -341,7 +340,7 @@ interface ChatClient {
         fun build(): ChatClient {
 
             if (apiKey.isEmpty()) {
-                throw ChatError("apiKey is not defined in " + this::class.java.simpleName)
+                throw IllegalArgumentException("apiKey is not defined in " + this::class.java.simpleName)
```

Comment:
Feels more like an `IllegalStateException` to me, as the Builder is in a state where `build` shouldn't be called on it

Reply:
I agree. Thanks